test/yargs.js

```diff
@@ -837,6 +837,17 @@ describe('yargs dsl tests', () => {
     })
   })
 
+  describe('parsed', () => {
+    it('should be false before parsing', () => {
```

Comment:
I wonder if we could figure out the specific exception that was happening with `showHelp` and put a test around this?

Reply:
I agree. A simple "showHelp should not fail" should do the job